### PR TITLE
Randomly Sampled CSV file created with script

### PR DESCRIPTION
**Pull request by dfamilia33**

### CODE REVIEW

File: webchild_sampling.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+from mowgli.lib.etl.pipeline_wrapper import PipelineWrapper
+from mowgli.lib.etl.web_child.web_child_pipeline import WebChildPipeline
+from mowgli.lib.etl.pipeline_storage import PipelineStorage
+
+
+
+def loadcsv():
+
+    print("Downloading Zip")
+    url = urlopen('http://people.mpi-inf.mpg.de/~ntandon/resources/relations/partOf/webchild_partof.zip')
```
Comment (gordom6) [inline webchild_sampling.py:19]: Use the extractor from the pipeline instead of downloading these yourself.

File: webchild_sampling.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        elif isinstance(node_or_edge, Edge):
+            edge = node_or_edge
+            if edge.predicate == WN_SYNSET:
+                continue
+            else:
+                part_whole_edges.append(edge)
+        else:
+            raise TypeError
+
+    selected_edges = sample(part_whole_edges, k= 200)
```
Comment (gordom6) [inline webchild_sampling.py:58]: The samples in the CSV are all related to anatomy, which seems improbable given the size of the WebChild data set.

File: webchild_sampling.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    with open('sampling.csv', 'w', newline='') as file:
+        writer = csv.writer(file)
+        writer.writerow(["Subject ID", "Subject word", "Subject definition","Object ID",
+         "Object Word", "Object Defnition","Predicate","Annotation"])
+
+        for el in selected_edges:
+            result = ["" for i in range(7)]
+            result[0]= el.subject
+            result[1]= nodes[el.subject].label
+            result[2]= definitions.get(el.subject) if definitions.get(el.subject) else 'None'
```
Comment (gordom6) [inline webchild_sampling.py:82]: definitions.get(el.subject, 'None')

File: webchild_sampling.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+    print('transforming')
+    triplegen = transformer.transform(memberof_csv_file_path ="webchild_partof_memberof.txt",
+        physical_csv_file_path="webchild_partof_physical.txt",
+        substanceof_csv_file_path="webchild_partof_substanceof.txt",
+        wordnet_csv_file_path="noun.gloss")
+
+
+    nodemap = dict()
+    edgemap = dict()
```
Comment (gordom6) [inline webchild_sampling.py:39]: Not used

File: webchild_sampling.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    transformer = WebChildTransformer()
+
+    print('transforming')
+    triplegen = transformer.transform(memberof_csv_file_path ="webchild_partof_memberof.txt",
+        physical_csv_file_path="webchild_partof_physical.txt",
+        substanceof_csv_file_path="webchild_partof_substanceof.txt",
+        wordnet_csv_file_path="noun.gloss")
+
+
+    nodemap = dict()
```
Comment (gordom6) [inline webchild_sampling.py:38]: Not used

File: webchild_sampling.py (hunk 1 above)
Comment (dfamilia33) [reply webchild_sampling.py:19]: Done, but for some reason I can't get permission to extract from the DATA_DIR so I placed the data in root.

Comment (gordom6) [reply webchild_sampling.py:19]: You should fix that problem instead of putting data in the root.

Comment (dfamilia33) [reply webchild_sampling.py:19]: It seemed to work after restarting VScode, I think it was a dependency issue with me manually deleting a folder within VScode.

File: webchild_sampling.py [hunk 6]
```diff
@@ -10,37 +10,41 @@
...
     transformer = WebChildTransformer()
 
-    print('transforming')
-    triplegen = transformer.transform(memberof_csv_file_path ="webchild_partof_memberof.txt",
-        physical_csv_file_path="webchild_partof_physical.txt",
-        substanceof_csv_file_path="webchild_partof_substanceof.txt",
-        wordnet_csv_file_path="noun.gloss")
+
+    print('Transforming')
+    triplegen = transformer.transform(memberof_csv_file_path =extraction.get("memberof_csv_file_path"),
```
Comment (gordom6) [inline webchild_sampling.py:36]: transform(**extraction)

File: webchild_sampling.py [hunk 7]
```diff
@@ -10,37 +10,41 @@
...
+from mowgli.paths import DATA_DIR, PROJECT_ROOT
+from os import listdir
 
 
 def loadcsv():
 
-    print("Downloading Zip")
-    url = urlopen('http://people.mpi-inf.mpg.de/~ntandon/resources/relations/partOf/webchild_partof.zip')
-    urlstream = BytesIO(url.read())
+    part_whole_zip_url = 'http://people.mpi-inf.mpg.de/~ntandon/resources/relations/partOf/webchild_partof.zip'
```
Comment (gordom6) [inline webchild_sampling.py:20]: The WebChildExtractor __init__ takes default parameters, you don't need to pass it any.

**Pull request merged**